Frameworks/Foundation/NSOperationQueue.mm

```diff
@@ -30,167 +30,320 @@
 #import "LoggingNative.h"
 #import "NSOperationInternal.h"
 
+#include <deque>
+#include <mutex>
+#include <condition_variable>
+#include <algorithm>
+#include <memory>
+
 static const wchar_t* TAG = L"NSOperationQueue";
 
-typedef void* gpointer;
+class NSOperationQueuePriv {
+public:
+    NSOperationQueuePriv(NSOperationQueue* parent) : _thread(nullptr), _maxConcurrentOperationCount(1), _isSuspended(NO) {
+        _name.attach([[NSString alloc] initWithFormat:@"NSOperationQueue %08x", parent]);
```

Comment:
I could but I don't think I will. Its a pretty long line already and I think it would look weird in the init list and it uses the weirder autorelease pattern that I don't super enjoy.

---
In reply to: [95449339](https://github.com/Microsoft/WinObjC/pull/1676#discussion_r95449339) [](ancestors = 95449339)